### DISCUSSION TITLE
Cannot read property "setComponentInstace" of undefined. Issue setting up project

### DISCUSSION THREAD

**Discussion by allihuwa** [jeffreylanters/react-unity-webgl#163] [Help Wanted!]
Hi, I am trying to link up my unity webgl project with this framework but I am having some problems. First I had to change the unityContext import for unityContent because UnityContext was not seen as a constructor. I found this solution here: 
https://github.com/elraccoone/react-unity-webgl/issues/118

But now I am getting a "TypeError: Cannot read property "setComponentIsntance" of undefined and am again stuck. Bellow is a picture of my vscode with the webgl build in the src/build folder and also the way I setup the app.js script. Can you help me identifying what is wrong or might be the problem? 
![webgl-react-issue](https://user-images.githubusercontent.com/15956953/110455977-f3f61900-80c0-11eb-9fd4-50d56c1c6b99.PNG)

Thank you

**Comment by jeffreylanters** [comment]
Hi. Which version of the module are you using?

**Comment by allihuwa** [comment]
I installed this version: react-unity-webgl@8.x because I am using unity version 2020.2.7f1

**Comment by jeffreylanters** [comment]
As of version 8.x of the module `UnityContext` should be used, could you try using this model instead?

**Reply by jeffreylanters** [reply]
Something like

```js
import React from "react";
import Unity, { UnityContext } from "react-unity-webgl"; // <-- UnityContext

const unityContext = new UnityContext({ // <-- UnityContext
  loaderUrl: "build/myunityapp.loader.js",
  dataUrl: "build/myunityapp.data",
  frameworkUrl: "build/myunityapp.framework.js",
  codeUrl: "build/myunityapp.wasm",
});

const App = () => {
  return <Unity unityContext={unityContext} />; // <-- unityContext
};
```

**Reply by saharsh-goyal** [reply]
I used this same model but still getting similar error `TypeError: window.createUnityInstance is not a function`
